src/pages/task/Metadata.tsx

```diff
@@ -30,6 +30,7 @@ export const Metadata: React.FC<{
   const baseCommit = get(task, "revision", "").slice(0, 10);
   const reliesOn = get(task, "reliesOn");
   const baseTaskMetadata = get(task, "baseTaskMetadata");
+  const ami = get(task, "ami");
```

Comment:
we can start using `task?.ami ?? ""` instead of `get` here. you could even replace all of these `get`s with optional chaining if you're up for it.

Reply:
Sure done.